db/structure.sql

```diff
@@ -218,7 +230,8 @@ CREATE TABLE pitchers (
     image_small character varying,
     image_large character varying,
     created_at timestamp without time zone NOT NULL,
-    updated_at timestamp without time zone NOT NULL
+    updated_at timestamp without time zone NOT NULL,
+    pitcher_type pitcher_type DEFAULT 'RP'::pitcher_type NOT NULL
```

Comment:
The default type is only so that the migration will pass. We shouldn't be defaulting new pitchers to relievers after the fact.